packages/ui/src/components/sign-in/sign-in.tsx

```diff
@@ -62,6 +63,7 @@ export function SignInComponent() {
                     <div className='space-y-4'>
                       <EmailField disabled={isGlobalLoading} />
                       <PasswordField disabled={isGlobalLoading} />
+                      <OTPField disabled={isGlobalLoading} />
```

Comment:
When tabbing into the OTPInput, the card gets offset. Likely due to the overflow width added to the OTP input to allow space for the one password icons.


https://github.com/clerk/javascript/assets/825855/361a69d6-d972-4f0d-9c8f-0736372db14a

Reply:
Should've fixed this 🤞